examples/custom_metrics/census/ops/upload_models.py

```diff
@@ -0,0 +1,72 @@
+from hydrosdk import Cluster, ModelVersion, LocalModel, DockerImage, SignatureBuilder
```

Comment:
Maybe we should also leave an example of serving.yaml  with monitoring and with application creation and how to use `hs apply`